src/GovUk.Education.ExploreEducationStatistics.Content.Model/Release.cs

```diff
@@ -66,6 +66,19 @@ public string ReleaseName
 
         public List<ReleaseStatus> ReleaseStatuses { get; set; }
 
+        public bool NotifySubscribers
+        {
+            get
+            {
+                return ReleaseStatuses.Any(rs => rs.ApprovalStatus == ReleaseApprovalStatus.Approved)
```

Comment:
I'm not sure what this is trying to achieve but it seems like "If there's been any approved status, then return NotifySubscribers of the last approved status, otherwise false".

That doesn't seem correct because a draft Release that's previously been approved with NotifySubscribers as true will return true but NotifySubscribers on the latest ReleaseStatus will be false as we know the service which sets it will force it to be false.

Seems like you might be trying to return the historical status of NotifySubscribers so that the checkbox remains ticked if it's previously been ticked, but that's not clear at all.

I would add some comments to explain what this is doing and add tests to ReleaseTests for this.

If it's simply intended to return the current value of NotifySubscribers then you could rewrite it as:

- Throw error if ReleaseStatuses is not hydrated
- Get the last status which would be LastOrDefault ReleaseStatus when ordered by Created
- Return `last?.NotifySubscribers ?? false`

Reply:
I think I've got the logic wrong around when notifications should be sent. And it seems that the code as it stands has bug potential, as there is no record of whether a notification has been sent. So we've decided to add a `NotifiedOn' column. This tells as whether a notification has been sent and provides protection against sending a notification twice.

Reply:
```
I'm not sure what this is trying to achieve but it seems like "If there's been any approved status, then return NotifySubscribers of the last approved status, otherwise false".```

To address this comment, my original thought on checking whether the ApprovalStatus was approved was to ensure that no future bugs were possible where an additional ReleaseStatus row was created between a release being scheduled for publication and actually being published. This could effect whether subscribers are notified, and potentially prevent them from being so because the code checks for the latest ReleaseStatus.